others/career-development.md

```diff
@@ -0,0 +1,73 @@
+# Career Development
+
+## CV
+
+### Best practice
+
+- Keep it short and concise, 1-2 pages maximum
+- Put your details at the top. Only these details are necessary:
+  - Name
+  - Email
+  - Phone number
+  - Your city (not your address since this is personal information! If they need it they'll ask for it later)
+- Use the following sections, and only populate them with relevant details starting from the most recent first
```

Comment:
Yeah on my CV I have Skills at the top but since I wasn't sure how conventional this was I put it in the bottom half since that was what was suggested in the Killer Resume link below. I'll put Skills at the top.

I also put Education above Employment against convention because I once read if you just graduated then it makes more sense to have Education first since that's what you most recently did and you don't have anything relevant to show in the Employment section. I'll switch this round.

I haven't heard of CVs without separate Education, Employment and Project sections. Is this format acceptable for employers?

Reply:
So - this is a good question, and I'm mostly basing it off my own expectations since I've had to read a reasonably large number of CVs, and been behind the table for maybe a couple of dozen interviews, mostly as the main interviewer, and I'm trying to think what would annoy me least about a non-standard CV. Nick might have a lot more experience than me so I'd like to get his input and I'm currently looking to try to find articles on career change CVs and mature degree CVs to see if there's more general advice.

The basic problem is that for most people in the world of work, the employment and education sections are laid out entirely chronologically. In the case of our students, they won't be. If we put employment first, then the whole first page is maybe not relevant, if we put education first, then we will have two chunks of education that may be separated by a big gap. That might be okay, and we might be sensibly advised to have the sections and order them Projects - Education - Employment. I have found it really annoying having to hunt about different places on the CV to determine if there were gaps, so we need to be a little careful there I think.

Reply:
https://www.brookes.ac.uk/students/careers/students-and-graduates/cv-guide/mature-graduate-cv/ has a linked sample CV which looks interesting. It takes the Skills - Qualifications - Employment route, and I don't think the education gap would be too distracting here if it weren't for some other iffy decisions. While summarising work experience is reasonable, having it split from the rest of the employment section by volunteering is a bit more problematic. It took me quite a few seconds to chain together the GCSEs finishing 1989 into the summary starting 1990 to the Nursing Auxiliary role starting 1993. I like that the career break is explained in employment.

There's a gap from 2001-2009 which is partly explained by volunteering one week per year 2002-2004, but is otherwise pretty weird, and would really benefit from an item similar to the career break item explaining why. Career gaps aren't necessarily deal-breakers, but as an interviewer you want to find out what happened - was your candidate in prison?

Reply:
Yeah having a CV laid out chronologically and ungrouped has the obvious benefit of avoiding jumping around to identify a sequential timeline. It would be good to get more input from others like Nick.

That mature CV is quite interesting. I've added it to the Career Development page.